.github/workflows/git-checks.yml

```diff
@@ -0,0 +1,12 @@
+name: Git Checks
+
+on: [pull_request]
+
+jobs:
+  block-fixup:
+    runs-on: ubuntu-18.04
```

Comment:
I would suggest you use the latest supported Ubuntu version

Reply:
It worked.